ProfessorAssessor/View Controllers/HomeViewController.h

```diff
@@ -1,11 +1,13 @@
 #import <UIKit/UIKit.h>
 #import "School.h"
+#import "Professor.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface HomeViewController : UIViewController
 
 @property (nonatomic, strong) School *school;
+@property (nonatomic, strong) Professor *searchedProfessor;
```

Comment:
Not really relevant where they came from. If there's only one profession in a type then it's just "professor"